core/src/prelude.rs

```diff
@@ -3,6 +3,7 @@ pub use crate::display_object::{
     DisplayObject, DisplayObjectContainer, HitTestOptions, TDisplayObject, TDisplayObjectContainer,
 };
 pub use ruffle_render::matrix::Matrix;
+pub use ruffle_render::matrix3d::Matrix3D;
```

Comment:
We don't actually use this as a prelude, so you don't need to add this line here

Reply:
True, thank you. Updated: [31c6e11](https://github.com/ruffle-rs/ruffle/pull/18810/commits/31c6e11e08c5ccd907bff829bb6fc5aaf18251a9)